terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/pom.xml

```diff
@@ -63,8 +63,8 @@
   <dependencies>
     <!-- == Begin BeanValidation == -->
     <dependency>
-      <groupId>javax.validation</groupId>
-      <artifactId>validation-api</artifactId>
+      <groupId>jakarta.validation</groupId>
+      <artifactId>jakarta.validation-api</artifactId>
```

Comment:
- [ ] I've only fixed pom.xml, is it okay if I don't fix the import statement?

Reply:
The Jakarta library we defined is Jakarta EE 8 compliant, and the package name has not changed from `javax.*`, so there is no need to modify the package name.The actual package for each library is as follows.
- [jakarta.el:jakarta.el-api:3.0.3](https://github.com/eclipse-ee4j/el-ri/tree/3.0.3-RELEASE/api/src/main)
- [jakarta.servlet.jsp:jakarta.servlet.jsp-api:2.3.6](https://github.com/eclipse-ee4j/jsp-api/tree/2.3.6/api/src/main)
- [jakarta.inject:jakarta.inject-api:1.0.3](https://github.com/eclipse-ee4j/injection-api/tree/1.0.3/src/main)
- [jakarta.validation:jakarta.validation-api:2.0.2](https://github.com/eclipse-ee4j/beanvalidation-api/tree/2.0.2/src/main)
- [jakarta.servlet:jakarta.servlet-api:4.0.4](https://github.com/eclipse-ee4j/servlet-api/tree/4.0.4-RELEASE/api/src/main)

In addition, considering that spring-boot uses Jakarta EE8 libraries (validation-api and servlet-api versions are defined in boot), I don't think there is any need to use Jakarta EE 9 on the terasoluna-gfw side.If you want to support Jakarta EE 9, you need to consider it separately.